include/fiction/algorithms/physical_design/ortho_ordering_network.hpp

```diff
@@ -0,0 +1,633 @@
+//
+// Created by benjamin on 11.04.23.
+//
+
+#ifndef FICTION_ORTHO_ORDERING_NETWORK_HPP
+#define FICTION_ORTHO_ORDERING_NETWORK_HPP
+
+#include "fiction/algorithms/network_transformation/inverter_substitution.hpp"
+#include "fiction/networks/views/input_ordering_view.hpp"
+#include "orthogonal.hpp"
+
+namespace fiction
+{
+
+namespace detail
+{
+
+template <typename Ntk>
+void paint_node_and_edges(const coloring_container<Ntk>& ctn, mockturtle::node<Ntk> const& n, uint32_t color)
+{
+    const auto finc = fanin_edges(ctn.color_ntk, n);
+    std::for_each(finc.fanin_edges.cbegin(), finc.fanin_edges.cend(),
+                  [&ctn, &color](const auto& fe) { ctn.color_ntk.paint_edge(fe, color); });
+    ctn.color_ntk.paint(mockturtle::node<Ntk>{n}, color);
+}
+
+template <typename Ntk>
+int64_t get_index(std::vector<Ntk> v, Ntk n)
```

Comment:
Please add a docstring. Also, what does this function do? It seems to look for a network in a vector of networks. Is `get_index` really a descriptive name for this behavior?

If you require an ordering of your networks to begin with, maybe `std::vector` is not the suitable container, depending on the number of networks you're processing.